src/kernel-install/90-loaderentry.install.in

```diff
@@ -138,6 +138,8 @@ mkdir -p "${LOADER_ENTRY%/*}" || {
 
 [ "$KERNEL_INSTALL_VERBOSE" -gt 0 ] && echo "Creating $LOADER_ENTRY"
 {
+    echo "# Boot Loader Specification entry"
+    echo "# File created by install.d/90-loaderentry.install (systemd {{GIT_VERSION}})"
```

Comment:
How about something like (not withstanding any changes needed to make this compliant with POSIX shell)?
```suggestion
    echo "# File created by $(realpath $0) (systemd {{GIT_VERSION}})"
```
This way the information would be more specific for the case somebody when somebody adds a drop in.

Reply:
hmm argv[0] might not be an absolute path...

Reply:
maybe `realpath /proc/self/exe`

Reply:
Just gave this a quick try and `realpath /proc/self/exe` just gives `/usr/bin/realpath` and doing `/proc/$$/exe` only gives the shell. Anyway, `realpath $0` would already solve the problem of `$0` not being absolute, but of course, `realpath` is not something one can really rely on always being there, so maybe this just isn't worth it.

Reply:
I adjusted the text. `$0` gives as exactly what we want, see the comment on the second commit.